m4/ax_check_openssl.m4

```diff
@@ -32,88 +32,93 @@
 #   and this notice are preserved. This file is offered as-is, without any
 #   warranty.
 
-#serial 9
+#serial 11
 
 AU_ALIAS([CHECK_SSL], [AX_CHECK_OPENSSL])
 AC_DEFUN([AX_CHECK_OPENSSL], [
     found=false
     AC_ARG_WITH([openssl],
         [AS_HELP_STRING([--with-openssl=DIR],
-            [With openssl support, see http://www.openssl.org])],
```

Comment:
This text shows up when  you do a ./configure --help; and the other ax_check_<lib>.m4 files have a similar description. Also, the short text spans over a single line. I do like to inform the user of the 'root' directory of the installation though. But in instances like SRP, the URL is actually quite informative to what library we actually mean.

If we change this here to append ", DIR=root of OpenSSL directory", we should probably do so for the other .m4 files.

Reply:
I'm reluctant to tamper with the upstream AX_CHECK_OPENSSL (autoconf-archive) macro downstream as it's not ours. But I take your point that the inconsistency is a bit lame.. hmm.